open_seq2seq/encoders/w2l_encoder.py

```diff
@@ -0,0 +1,187 @@
+# Copyright (c) 2018 NVIDIA Corporation
+from __future__ import absolute_import, division, print_function
+from __future__ import unicode_literals
+from six.moves import range
+
+import tensorflow as tf
+
+from .encoder import Encoder
+
+def conv_bn_actv(layer, name, inputs, filters, kernel_size, activation_fn, strides,
```

Comment:
I think it would be better to reuse this function across speech2text encoders. Can you add if statement for the bn dimensions: if dimension is not 4, do the expand dims and squeeze) and than share this function across w2l and ds2 encoders, either by importing it from one of the encoders files or by making additional utils.py file.

Reply:
Sure! Do you think its better to add a new part cnns in 'open_seq2seq/parts' which has this function along with other potential functions in the future like 'conv_wn_actv'?